src/main.rs

```diff
@@ -13,6 +13,7 @@ use std::{fs, io};
 use term_painter::Attr::*;
 use term_painter::Color::*;
 use term_painter::ToStyle;
+use parse_duration::parse;
```

Comment:
Since there's a lot of parsing going on in this project already, I think it would be better to either rename this as `parse_duration` or just call it below as `parse_duration::parse` to avoid any ambiguity

Reply:
Naming the function same as the crate made the compiler unhappy and required too much ceremony so I went with renaming it to parse_systemd_time which is explicit in terms of what it's intended to parse.